Resources/Prototypes/Recipes/Construction/Graphs/machines/machine.yml

```diff
@@ -98,3 +103,18 @@
             - tool: Prying
               doAfter: 0.25
 
+    - node: destroyedMachineFrame
+      entity: MachineFrameDestroyed
+      edges:
+        - to: start
+          steps:
+            - tool: Welding
+              doAfter: 5            
+          completed:
+            - !type:SpawnPrototype
+              prototype: SheetSteel1
+              amount: 5
+            - !type:SpawnPrototype
+              prototype: CableHVStack1
+              amount: 2
```

Comment:
I'm not sure why a destroyed frame is constructible in the first place, but currently there's an issue where it takes only 5 steel sheets to create, but leaves behind 5 sheets & 2 HV wires when deconstructed. It should either not be constructible, not leave behind HV wires, or both.

Reply:
> 

Currently the only thing using the destoyed machine frame is the substation and I wanted it to drop some HV cables (whereas everything else is destroyed in the explosion). I can just remove that and say only the steel is salvegable.

The frame itself is not actually constructible but in order for deconstruction you need a complete graph from start to end. Same as my AME deconstruction PR.